mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -497,8 +457,12 @@ impl UtxoRpcClientOps for NativeClient {
 
     fn display_balance(&self, address: Address, _decimals: u8) -> RpcRes<BigDecimal> {
         Box::new(
-            self.list_unspent(0, std::i32::MAX, vec![address.to_string()])
-                .map(|unspents| unspents.iter().fold(0., |sum, unspent| sum + unspent.amount).into()),
+            self.list_unspent_impl(0, std::i32::MAX, vec![address.to_string()])
+                .map(|unspents| {
+                    unspents.iter().fold(BigDecimal::from(0), |sum, unspent| {
+                        &sum + BigDecimal::from_str(&unspent.amount.to_string()).unwrap()
```

Comment:
Amount is MmNumber here so this can be changed to just `unspent.amount.to_decimal()`. This code left in intermediate stage after I first tried to use the JSON RawValue for amount.